Sources/Private/CoreAnimation/Animations/DropShadowAnimation.swift

```diff
@@ -0,0 +1,183 @@
+  var _opacity: KeyframeGroup<LottieVector1D>? { get }
+
+  /// The shadow radius of the blur
+  var _radius: KeyframeGroup<LottieVector1D>? { get }
+
+  /// The color of the drop shadow
+  var _color: KeyframeGroup<LottieColor>? { get }
+
+  /// The angle of the drop shadow, in degrees,
+  /// with "90" resulting in a shadow directly beneath the layer.
+  /// Combines with the `distance` to form the `shadowOffset`.
+  var _angle: KeyframeGroup<LottieVector1D>? { get }
+
+  /// The distance of the drop shadow offset.
+  /// Combines with the `angle` to form the `shadowOffset`.
+  var _distance: KeyframeGroup<LottieVector1D>? { get }
+}
+
+// MARK: - DropShadowStyle + DropShadowModel
+
+extension DropShadowStyle: DropShadowModel {
+  var _opacity: KeyframeGroup<LottieVector1D>? { opacity }
+  var _color: KeyframeGroup<LottieColor>? { color }
+  var _angle: KeyframeGroup<LottieVector1D>? { angle }
+  var _distance: KeyframeGroup<LottieVector1D>? { distance }
+
+  var _radius: KeyframeGroup<LottieVector1D>? {
+    size.map { sizeValue in
+      // `DropShadowStyle.size` is approximately double as large
+      // as the visually-equivalent `cornerRadius` value
+      LottieVector1D(sizeValue.cgFloatValue / 2)
+    }
+  }
+}
+
+// MARK: - DropShadowEffect + DropShadowModel
+
+extension DropShadowEffect: DropShadowModel {
+  var _color: KeyframeGroup<LottieColor>? { color?.value }
+
+  var _distance: KeyframeGroup<LottieVector1D>? {
+    distance?.value?.map { distanceValue in
+      // `DropShadowEffect.distance` doesn't seem to map cleanly to
+      // `CALayer.shadowOffset` (e.g. with a simple multiplier).
+      // Instead, this uses a custom quadratic regression eyeballed
+      // to match the expected appearance of the start / end of the
+      // `issue_1169_shadow_effect_animated.json` sample animation:
+      //  - `distance=5` roughly corresponds to an offset value of 4
+      //  - `distance=10` roughly corresponds to an offset value of 5
+      // This could probably be improved with more examples.
+      let x = distanceValue.cgFloatValue
+      let cornerRadiusMapping = (-0.06 * pow(x, 2)) + (1.1 * x)
+
+      return LottieVector1D(cornerRadiusMapping)
+    }
+  }
+
+  var _radius: KeyframeGroup<LottieVector1D>? {
+    softness?.value?.map { softnessValue in
+      // `DropShadowEffect.softness` doesn't seem to map cleanly to
+      // `CALayer.cornerRadius` (e.g. with a simple multiplier).
+      // Instead, this uses a custom quadratic regression eyeballed
+      // to match the expected appearance of the start / end of the
+      // `issue_1169_shadow_effect_animated.json` sample animation:
+      //  - `softness=10` roughly corresponds to `cornerRadius=2.5`
+      //  - `softness=50` roughly corresponds to `cornerRadius=6.25`
+      // This could probably be improved with more examples.
+      let x = softnessValue.cgFloatValue
+      let cornerRadiusMapping = (-0.003 * pow(x, 2)) + (0.281 * x)
```

Comment:
Worked with Remington to get this to match the preview in After Effects. For example:

| Start, iOS | End, iOS |
| ---- | ---- |
| <img width="745" alt="Screenshot 2023-08-15 at 3 22 13 PM" src="https://github.com/airbnb/lottie-ios/assets/1811727/45494433-a047-4dac-b4df-106f9e6d1261"> | <img width="745" alt="Screenshot 2023-08-15 at 3 22 16 PM" src="https://github.com/airbnb/lottie-ios/assets/1811727/39cd653c-5b19-484f-ae83-50024d9e996a"> |

| Start, After Effects | End, After Effects |
| ---- | ---- |
| ![Screenshot 2023-08-15 at 11 01 30 AM](https://github.com/airbnb/lottie-ios/assets/1811727/234f3ff0-a82b-4d6a-a9d2-2f97bbb3299e) | ![Screenshot 2023-08-15 at 11 01 36 AM](https://github.com/airbnb/lottie-ios/assets/1811727/fd12d2a0-9632-4bdd-9eec-bd91795ebff4) |

This was the only mapping I could find that we thought looked right in both of these cases